worlds/hk/docs/setup_en.md

```diff
@@ -1,15 +1,32 @@
 # Hollow Knight for Archipelago Setup Guide
 
 ## Required Software
-* Download and unzip the Scarab Mod Manager from the [Scarab GitHub Releases page](https://github.com/fifty-six/Scarab/releases).
-* A legal copy of Hollow Knight, not purchased or played through XBox Game Pass.
-    * Unfortunately, the Game Pass version is not currently compatible with mods.
+* Download and unzip the Scarab+ Mod Manager from the [Scarab+ website](https://themulhima.github.io/Scarab/).
+* A legal copy of Hollow Knight.
+
+## Optional Software
+* Archipelago Map Mod from Scarab+
 
 ## Installing the Archipelago Mod using Scarab
-1. Launch Scarab and ensure it locates your Hollow Knight installation directory.
-2. Click the "Install" checkbox near the "Archipelago" mod entry.
+1. Launch Scarab+ and ensure it locates your Hollow Knight installation directory.
+2. Click the "Install" button near the "Archipelago" mod entry.
```

Comment:
Not really super relevant to the particular change being made here but should we also mention the map mod somewhere in here? If so we should also be sure to mention that RandoMapMod and MapChanger should be uninstalled or disabled if coming from the solo rando (maybe that's a point for the map mod readme?)

Reply:
I added it under a new "Optional Software" section. I think a warning about incompatible mods wouldn't hurt there